### ISSUE TITLE
Latest version in production?

### ISSUE THREAD

**Issue by Potherca** [solid-contrib/catalog#3]
There is a typo in the UI, "Applictions" is missing an _a_.

![image](https://github.com/user-attachments/assets/944d16e0-d9ab-4b54-aa29-b531aad980d2)

This seems to have been fixed in the repo here:

https://github.com/solid-contrib/catalog/commit/4cb5e6d91fe2d94eb49a86c6b7718a19824dc108#diff-29c7b4a1584ea9cefd8b9b40b4ebe3d286091b33d192d09d7ecc545beb6d42d3L157

Maybe the site has not been updated?

**Comment by jeff-zucker** [comment]
Sorry, I'm behind schedule and haven't figured out how I want to sync between github and the pod so, as of now, you are right, they are out of sync.

**Comment by Potherca** [comment]
Should we start thinking about automation, or is that still too early?

**Comment by jeff-zucker** [comment]
> Should we start thinking about automation, or is that still too early?

re automating the upload of the data -  I am investigating github actions to automate posting to the pod on merge, need to set up the repos first (this repo will be for the app, the other repo will be for the data).

re automating the catalog itself - I will be uploading a roadmap and other notes this week.  Automation is definitely in the plan but I think we need to pin down the shape and get it working as a static RDF first.  Later today I'm going to upload my proposal to use an ADMS/DCAT profile to define the shape and structure  of the catalog.  ADMS is specifically designed to work with aggregation.

**Comment by elf-pavlik** [comment]
I assume the automation would be to deploy the catalog **app** automatically whenever the main branch changes.

I also understood that all the catalog **data** work has moved to https://github.com/solid/catalog/

**Comment by jeff-zucker** [comment]
Yes, correct, the automation would be to post changes from both repos to the pod on merge.  The RDF, SHACL, and SKOS files will be uploaded from  https://github.com/solid/catalog/ and the JavaScript, HTML, and CSS will be uploaded from https://github.com/solid-contrib/catalog/ and all will end up in https://solidproject.solidcommunity.net/catalog/.   That will allow separation of the app from the data and structure.
